articles/anomaly-detection/index.md

```diff
@@ -16,82 +16,99 @@ useCase: customize-anomaly-detection
+Another trigger occurs if there are 50 sign up attempts per minute from the same IP address.
 
-Auth0 does email the dashboard administrator(s) when this block is triggered. Within this email there's a link the owner can click on to remove the block.
+#### Actions
 
-#### Enable or Disable Brute Force Protection
+* Notify dashboard administrator(s).
+* Block suspicious addresses.
 
-By default, brute force protection is enabled for all connections.
+If this block is triggered, additional access attempts are released one-at-a-time over the course of 24 hours until 100 attempts are allocated. This results in approximately 1 additional attempt every 15 minutes.
 
-Each connection has a flag called `brute_force_protection` that you can use to disable brute force protection. If this flag is set to `true`, then brute force protection is enabled *even if general brute force protection is enabled*.
+Auth0 emails the dashboard administrator(s) when this block is triggered. Within this email there's a link the owner can click on to remove the block.
 
-We do not recommend setting the `brute_force_protection` flag to `false` (effectively disabling brute force protection for the connection), but if you do, you will be able to change this in the Dashboard. There will be a **Improve brute force protection** toggle under Connection Settings that changes the flag from `false` to `true`.
-
-#### Restrictions Regarding Brute-Force Protection
+### Restrictions Regarding Brute-Force Protection
 
 Both of these anomaly types depend on the IP address of the user. Because of this, the following use cases are *not* supported:
 
 1. Using the [Resource Owner](/api/authentication#resource-owner) from the backend of the application. Using this call does not get the IP address of the user. See point 2 below as an alternative.
 2. Using [Resource Owner Password Grant](/api-auth/grant/password) from the backend of the application. Using this call does not get the IP address of the user, however, you can [configure your application and send the IP address of the user as part of the request](/api-auth/tutorials/using-resource-owner-password-from-server-side) to make brute-force protection work correctly.
 3. Authenticating many users from the same IP address. For example, users that are behind a proxy are more likely to reach these limits and trigger the associated protection. It is possible to configure a whitelist for the proxy's IP and CIDR range and avoid erroneously triggering the protection.
 
-### Breached Password Detection
+### Enable or disable Brute Force Protection
+
+Brute force protection is enabled for all connections by default.
+
+Each connection has a flag called `brute_force_protection` that you can use to disable brute force protection. If this flag is set to `true`, then brute force protection is enabled *even if general brute force protection is enabled*.
```

Comment:
This setting is no longer available and should not be used. It exists purely for backward compat reasons in some connections AFAIK.

Reply:
Deleted